### PR TITLE
Few changes. 

### PR DESCRIPTION
**Pull request by vinc3nz**
Few changes, may require short testing. No breaking changes.

### CODE REVIEW

**Review by DIBSTERYT — APPROVED** [review]
Everything else looks good. Will be up to Dan to merge such changes. Recommend pushing one more commit to disable the minecraft command.

**Review by danielpmc — APPROVED** [review]

**Comment by DIBSTERYT** [comment]
@danielpmc You alright with merging?

**Comment by danielpmc** [comment]
Sure. Bot won't update automatically, though, as there are local changes not yet pushed to GH

**Comment by danielpmc** [comment]
Sure. Bot won't update automatically, though, as there are local changes not yet pushed to GH

File: bot/discord/commands/server/list.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const axios = require('axios')
 exports.run = async(client, message, args) => {
-    message.channel.send('Loading servers...')
+    const loadingMsg = message.channel.send('Loading servers...')
```
Comment (itzJOHv) [inline bot/discord/commands/server/list.js:3]: `await` is required here

Comment (DIBSTERYT) [reply bot/discord/commands/server/list.js:3]: Just fixed this. 👍 

https://github.com/DanBot-Hosting/DanBotHostingStats/commit/fe9cd8951bd1e39c8334c9ed352301202dbbf696

**Comment by Microwave-6443** [comment]
ig `.toLocaleString()` went out the window for the `arr.length` stuff

**Pull request merged**